src/graphql/middleware/auth.ts

```diff
@@ -0,0 +1,28 @@
+import { PrismaClient } from ".prisma/client";
+import { MyContext } from "../../types/MyContext";
+import { verifyToken } from "../../utils/authHelpers/verifyToken";
+import { MiddlewareFn } from "type-graphql";
+
+const prisma = new PrismaClient();
+
+export const isAuth: MiddlewareFn<MyContext> = async ({ context }, next) => {
+	if (!context.req.headers.authorization) {
+		throw new Error("Add authoriation");
+	}
+
+	const token = context.req.headers.authorization;
+
+	const userId = await verifyToken(token);
+
+	const employeeId = await prisma.employee.findUnique({
```

Comment:
Prisma should only be used by services. In this case, you should use the "getEmployeeById" method that exists in your employeeService. If it doesn't exist then create it.
 
 Rule of thumb. Never use prisma outside the services folder.

Reply:
Also you can remove the spacing between userId and employeeId

const userId = 
const employeeId =